lib/rules/space-in-parens.js

```diff
@@ -184,9 +188,9 @@ module.exports = function(context) {
 
     return {
         "Program": function checkParenSpaces(node) {
-            var tokens = node.tokens;
-            var prevToken, nextToken;
+            var tokens, prevToken, nextToken;
             exceptions = getExceptions();
+            tokens = sourceCode.tokensAndComments;
```

Comment:
Yup, my point is that introduces overhead that is unnecessary because the value never changes. It's a lot less expensive to access a property than to call a function.